docs/modules/phenopackets_api.rst

```diff
@@ -0,0 +1,332 @@
+
+- datasets (single or multiple list of datasets titles separated by comma):
+  :code:`/api/variants?datasets=dataset_1,dataset_2`
+
+- authorized_datasets (single or multiple list of authorized datasets titles separated by comma):
+  :code:`/api/variants?authorized_datasets=dataset_1,dataset_2`
+
+
+**Diseases**
+
+:code:`api/diseases` GET: list of Diseases
+
+:code:`api/diseases/{id}` GET: single Disease
+
+The following **filters** can be applied:
+
+- id (single, exact match), disease id in Katsu database: :code:`/api/diseases?id=1`
+
+- term (case-insensitive partial match): :code:`/api/diseases?term=COVID-19`
+  or :code:`/api/diseases?term=SNOMED:840539006`
+
+- extra_properties (case-insensitive partial match): :code:`/api/diseases?extra_properties=test`
+
+- extra_properties_datatype (ONLY if "datatype" is present in extra_properties, case-insensitive partial match):
+  :code:`/api/diseases?extra_properties_datatype=comorbidities`
+
+- extra_properties_comorbidities_group (ONLY if "comorbidities_group" is present in extra_properties, case-insensitive partial match):
+  :code:`/api/diseases?extra_properties_comorbidities_group=common`
+
+- datasets (single or multiple list of datasets titles separated by comma):
+  :code:`/api/diseases?datasets=dataset_1,dataset_2`
+
+- authorized_datasets (single or multiple list of authorized datasets titles separated by comma):
+  :code:`/api/diseases?authorized_datasets=dataset_1,dataset_2`
+
+
+**Biosamples**
+
+:code:`api/biosamples` GET: list of Biosamples
+
+:code:`api/biosamples/{id}` GET: single Biosample
+
+The following **filters** can be applied:
+
+- id (single, exact match): :code:`/api/biosamples?id=1`
+
+- description (case-insensitive partial match): :code:`/api/biosamples?description=test`
+
+- sampled_tissue (case-insensitive partial match): :code:`/api/biosamples?sampled_tissue=urinary bladder`
+  or :code:`/api/biosamples?sampled_tissue=UBERON:0001256`
+
+- taxonomy (case-insensitive partial match): :code:`/api/biosamples?taxonomy=homo sapiens`
+  or :code:`/api/biosamples?taxonomy=NCBITaxon:9606`
+
+- histological_diagnosis (case-insensitive partial match): :code:`/api/biosamples?histological_diagnosis=negative finding`
+  or :code:`/api/biosamples?histological_diagnosis=NCIT:C38757`
+
+- tumor_progression (case-insensitive partial match): :code:`/api/biosamples?tumor_progression=primary neoplasm`
+  or :code:`/api/biosamples?tumor_progression=NCIT:C8509`
+
+- tumor_grade (case-insensitive partial match): :code:`/api/biosamples?tumor_grade=healed`
+  or :code:`/api/biosamples?tumor_grade=NCIT:C41133`
+
+- individual (single, exact match, biosample must be related to Individual via ForeignKey not via Phenopacket):
+  :code:`/api/biosamples?individual=10001`
+
+- procedure (single, exact match, searches by procedure id): :code:`/api/biosamples?procedure=1`
+
+- is_control_sample: :code:`/api/biosamples?is_control_sample=No`
```

Comment:
Same as other boolean fields: works with true/false, not with `Yes`/`No`/`Unknown`

Reply:
fixed